README.md

```diff
@@ -94,8 +98,8 @@ these data correspond to newer data gathering techniques (primarily LIDAR).
 
 This reference data corresponds to a snapshot of the latest USGS data available from July 2017.
 
-The data is read by the NED terrain driver found in `harness/src/reference_models/geo/terrain.py`, 
-and in particular is used by the reference propagation models in `harness/src/reference_models/propagation`.
+The data is read by the SAS NED terrain driver found in `harness/src/reference_models/geo/terrain.py`, 
+and in particular is used by the SAS reference propagation models in `harness/src/reference_models/propagation`.
 
```

Comment:
Note that I will either copy these drivers into AFC, or provide a common version in the Common-Data folder as a reference. But dont worry, I will update the doc as I do.